### ISSUE TITLE
Cannot run nrpe check

### ISSUE THREAD

**Issue by OskarKjellin** [zalando-zmon/zmon-worker#163]
It seems like from #29 that it was proposed to remove nrpe from the default plugins, but I cannot really understand how it's intended to work.

In the docker container it does not work and on https://demo.zmon.io it does not.
It just returns 
`"[Errno 2] No such file or directory: '/usr/lib/nagios/plugins/check_nrpe'"`

**Comment by hjacobs** [comment]
@OskarKjellin actually it will not work right now with the Docker image even if you enable the plugin --- simply because nothing installs the `check_nrpe` scripts (you need your own Nagios/Icinga nrpe scripts for this).

**Comment by OskarKjellin** [comment]
Hmm, that's too bad. Why isn't the agent needed to run the plugin included?

**Comment by Jan-M** [comment]
Using Docker there is little that prevents you from quickly putting together another image based on our latest worker adding scripts and setup required for nrpe in your environment.

We will see how we will handle this in the future, depending on the needs reported.

**Comment by OskarKjellin** [comment]
Yes, ofc I can create my own docker image, just nice when you don't have to maintain a clone to install a needed dependency. 

Not sure but it feels like we're talking about different use cases. I want the worker to execute checks that are configured on a remote host. So basically my Dockerfile would be:

```
FROM registry.opensource.zalan.do/stups/zmon-worker:cd198
RUN apt-get install nagios-nrpe-plugin -y
```

Doing that got rid of the error. It kinda feels like the plugin assumes that I want to execute them towards the localhost however, which is not how we use it (we generally use the -H parameter to check_nrpe)
